.github/workflows/release-publish.yml

```diff
@@ -27,6 +27,11 @@ jobs:
           git config user.name "$GITHUB_ACTOR"
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
+      - name: Import PGP Private Key
+        run: |
+          echo "${{ secrets.PGP_PRIVATE_KEY }}" | gpg --dearmor --output keyring.gpg
+          echo "${{ secrets.PGP_KEY_PASSPHRASE }}" > passphrase-file.txt
```

Comment:
maybe passing it through the env would be better, see https://github.com/keptn/lifecycle-toolkit-charts/blob/3de78103af8f8f3690c3a0106508c036a50ac56a/.github/workflows/release.yml#L40C1-L42C1

Reply:
I now noticed in the example above they are also creating it as a file and just passing the filepath... nevermind 😅